packages/cli/src/commands/extensions/update.ts

```diff
@@ -92,14 +89,18 @@ export async function handleUpdate(args: UpdateArgs) {
   if (args.all) {
     try {
       const extensionState = new Map();
-      await checkForAllExtensionUpdates(extensions, (action) => {
-        if (action.type === 'SET_STATE') {
-          extensionState.set(action.payload.name, {
-            status: action.payload.state,
-            processed: true, // No need to process as we will force the update.
-          });
-        }
-      });
+      await checkForAllExtensionUpdates(
+        extensions,
+        (action) => {
+          if (action.type === 'SET_STATE') {
+            extensionState.set(action.payload.name, {
+              status: action.payload.state,
+              notified: true, // No need to process as we will force the update.
```

Comment:
update the comment to align with the code.
Also this is now a lie as we haven't notified.
to be overly pedantic we could label this
notificationRequested or something similar. Then it can be false here to exactly indicate the state.

Reply:
I could also just remove this entirely, its not necessary in this code path, and this code path does not do any notifications of available updates, it always installs them.

Reply:
I went ahead and just made that change, to just not indicate we notified. This code path is for the regular update command outside the CLI itself.